docs/developer/architecture/logging.rst

```diff
@@ -0,0 +1,235 @@
+Logging
+=======
+
+This section summarizes centralized logging design architecture which is closely
+linked to the above requirements (to be merged).
+
+The following terms will be used throughout the context of this document so it
```

Comment:
Maybe we're missing one key component: the Database (or Storage, whichever name fits).
It would help to "see" the pipeline (ASCII schema can be sufficient):

```
Collector -> Aggregator -> Database -> Visualizer
```

Each component can be scaled and scheduled independently.

Reply:
Also, in this document, it's important to consider these "components" and the split between them as **part of the design**. Nothing forces us to think of this log centralization system in this manner, it's a choice to match our requirements.